au4/src/projectscene/qml/Audacity/ProjectScene/trackspanel/AddNewTrackPopup.qml

```diff
@@ -18,170 +18,49 @@ StyledPopupView {
-        property bool validNumberOfTracks: true
-    }
+    signal createTrack(type : int)
 
     ColumnLayout {
         id: content
 
         spacing: 12
 
-        RadioButtonGroup {
-            id: trackType
+        RowLayout {
 
             width: parent.width
             height: 64
-
-            readonly property int navigationRowStart: 0
-            readonly property int navigationRowEnd: navigationRowStart + count
-
-            model: [
-                { iconCode: IconCode.MICROPHONE, text: qsTrc("projectscene", "Mono"), value: TrackType.MONO },
-                { iconCode: IconCode.MICROPHONE, text: qsTrc("projectscene", "Stereo"), value: TrackType.STEREO },
-                { iconCode: IconCode.LOOP_IN, text: qsTrc("projectscene", "Label"), value: TrackType.LABEL },
-            ]
-
-            delegate: FlatRadioButton {
-                width: 97
-                height: 64
-
-                Column {
-                    anchors.centerIn: parent
-                    height: childrenRect.height
-                    spacing: 8
-
-                    StyledIconLabel {
-                        anchors.horizontalCenter: parent.horizontalCenter
-                        iconCode: modelData.iconCode
-                    }
-
-                    StyledTextLabel {
-                        anchors.horizontalCenter: parent.horizontalCenter
-                        text: modelData.text
-                    }
-                }
-
-                checked: prv.type === modelData.value
-                onToggled: {
-                    prv.type = modelData.value
-                }
-            }
-        }
-
-        StyledTextLabel {
-            id: numberOfTracksLabel
-
-            Layout.alignment: Qt.AlignHCenter
-            verticalAlignment: Text.AlignVCenter
-            horizontalAlignment: Text.AlignHCenter
-
-            text: prv.numberOfTracksText
-            font.family: ui.theme.bodyFont.family
-        }
-
-        RowLayout {
-
-            Layout.alignment: Qt.AlignHCenter
-
-            FlatButton {
-                id: subtract
-
-                Layout.preferredWidth: 28
-                Layout.preferredHeight: 28
-
-                text: "-"
-
-                onClicked: {
-                    if (prv.numberOfTracks === "") {
-                        input.focus = false
-                        return
+            spacing: 12
+
+            Repeater {
+                model: [
+                    { type: TrackType.MONO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Mono") },
+                    { type: TrackType.STEREO, icon: IconCode.MICROPHONE, text: qsTrc("projectscene", "Stereo") },
+                    { type: TrackType.LABEL, icon: IconCode.LOOP_IN, text: qsTrc("projectscene", "Label") }
+                ]
+
+                FlatButton {
+                    Layout.preferredWidth: 64
+                    Layout.preferredHeight: 64
+                    Layout.fillHeight: true
```

Comment:
please add empty lines between property groups, it helps in code readability
see https://github.com/musescore/MuseScore/wiki/CodeGuidelines#qml

Reply:
Will address after rebase and before merge, will save a build run.